ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-ruby27-v1.14.x.yaml

```diff
@@ -52,7 +52,7 @@ base_images:
     namespace: logging
     tag: logging-elasticsearch6
   logging-eventrouter:
-    name: 5.y
+    name: 6.y
```

Comment:
same comment here.  I guess 5.y was pruned because there are no jobs referencing it any longer :thinking:

Reply:
ref: https://github.com/ViaQ/logging-fluentd

Reply:
For me, it looks as if images get pruned if nothing is _producing_ them anymore. I guess this is an effort to keep the number of images small and also to avoid CI running with stale images that are not being updated anymore.